src/components/EpisodePreview/EpisodePreview.jsx

```diff
@@ -0,0 +1,93 @@
+import React, { useContext, useState } from "react";
+import {
+	EpisodesContext as Episodes
+} from "COMPONENTS/Episodes";
+
+export default function EpisodePreview(props){
+
+	// CONTEXT
+	//-----------------------------
+	const {
+		activeIndex,
+		episodeCount
+	} = useContext(Episodes);
+
+
+	// STATE
+	// ----------------------------
+	const [hidden, setHidden] = useState(true);
+
+
+	// RENDER
+	// ----------------------------
+	const { 
+		index,
+		number,
+		title,
+		thumbSrc,
+		thumbDesc,
+		videoSrc
+	} = props;
+
+	const previewId = `episode__${number}__preview`;
+
+	const prevIndex = index - 1;
+	const nextIndex = index + 1;
+
+	let prevLink, nextLink;
+
+	if(prevIndex > -1){
+		prevLink = 
+			<a href={ `#episode__${number-1}__preview` }>
+				Prev
+			</a>
+		;
+	}
+
+	if(nextIndex < episodeCount){
+		nextLink =
+			<a href={ `#episode__${number+1}__preview` }>
```

Comment:
just spotted this - if `number` is an index then we should keep it called `index` to indicate that it's zero-indexed. If I saw `episode number 7` as a concept then I would assume it's the 7th, whereas `episode index 7` would be the 6th without needing to look anything up. If you wanna use `number` then the whole `+1` stuff should happen before that and have `const episodeNo = index + 1` or something.

Reply:
To clarify: number is not zero-indexed. 
Good feedback though. I've renamed as episodeNo in the destructure, set up 2 new vars called prevEpisodeNo and nextEpisodeNo and used them instead to make it cleaner.